### PR TITLE
AAP-20343: Created docs for post GA bot improvements feature

### PR DESCRIPTION
**Pull request by sayjadha**
This PR addresses JIRA https://issues.redhat.com/browse/AAP-20343.

Changes are as follows:

- Post-GA bot improvement updates
- Release notes updates to the release notes

### CODE REVIEW

**Review by EMcWhinn — COMMENTED** [review]
Few suggestions, otherwise LGTM.

File: lightspeed/assemblies/assembly_using-code-bot-for-suggestions.adoc [hunk 1]
```diff
@@ -36,17 +39,16 @@ The following examples are code recommendations that the {AnsibleCodeBot} can su
...
 
-* If there is an existing pull request that is not on the latest commit default branch, Ansible code bot does a pull request difference to compare the changes in both the branches. The following scenarios are possible:
-** *There is no difference in the existing and new scan results*: Ansible code bot does not push the scan results as a new pull request. 
-** *There are differences found in the existing and the new scan results*: {AnsibleCodeBot} creates a new pull request. The newly-created pull request does not close the existing pull request, against which the pull request difference was noted. This behavior makes it easier for the repository administrator to review only the latest pull request created by the {AnsibleCodeBot}, and the administrator can avoid reviewing the older pull requests created by the bot. If required, the administrator can close the older pull requests.
+* If there is an existing pull request that is not on the latest commit default branch, the {AnsibleCodeBot} does a pull request difference to compare the changes in both the branches. The following scenarios are possible:
```
[the remark this answers is not in the document]
Comment (sayjadha) [reply lightspeed/assemblies/assembly_using-code-bot-for-suggestions.adoc:49]: Made the change.

File: lightspeed/modules/proc_install-code-bot.adoc [hunk 2]
```diff
@@ -2,34 +2,47 @@
...
 
-[role="_additional-resources"]
-.Additional resources
-* xref:configure-repo-scan_using-code-bot-for-suggestions[Configuring Ansible code bot to scan your repository at regular intervals]
-* xref:manually-scan-repo_using-code-bot-for-suggestions[Manually scanning your Git repositories]
-* xref:troubleshooting-code-bot_troubleshooting-lightspeed[Troubleshooting Ansible code bot errors]
+.Procedure
+. Log in to GitHub by using the account associated with your organization.
+. In GitHub, click your profile photo > *Settings*.  
+. nder Integrations, click *Applications* > *Installed GitHub Apps*.
```
[the remark this answers is not in the document]
Comment (sayjadha) [reply lightspeed/modules/proc_install-code-bot.adoc:44]: Good catch, made the change.

File: lightspeed/modules/proc_manually-scan-repo.adoc [hunk 3]
```diff
@@ -4,32 +4,45 @@
...
-If the avoid duplicate PRs condition is not met, then a manual scan results in a new PR with all the necessary Ansible code bot recommendations.
+== Manually scanning the repository from the {AnsibleCodeBot}  dashboard
 
-Following are the avoid duplicate conditions: 
+.Procedure
+. Log in to the link:https://bot.ai.ansible.redhat.com/console[{AnsibleCodeBot} dashboard]. 
++
+The *Repositories* list displays a list of repositories that you selected for scanning. 
++
+NOTE: If you do not see your repository in the *Repositories* list, you can add it for scanning. For more information, see xref:add-remove-repo-from-scans_using-code-bot-for-suggestions[Adding or removing repositories from the {AnsibleCodeBot}].
```
[the remark this answers is not in the document]
Comment (sayjadha) [reply lightspeed/modules/proc_manually-scan-repo.adoc:16]: @EMcWhinn : The syntax in the doc follows the "Example 1. Admonition paragraph syntax" per the Asciidoc syntax, without the callouts. Not sure what change is required here.

File: lightspeed/modules/proc_manually-scan-repo.adoc [hunk 4]
```diff
@@ -4,32 +4,45 @@
...
+. To start a manual scan of your repository, click the *Ellipsis* icon (image:ansible-code-bot-dashboard-kebab-icon.png[Ellipsis icon]) beside the repository that you want to scan and select *Scan now*.
 
-.Procedure
+. Click *Refresh* to view the status of the scan job.
 
+. To view more details about your repository scans, click the *Ellipsis* icon (image:ansible-code-bot-dashboard-kebab-icon.png[Ellipsis icon]) beside the repository and select *View scan history*.
++
+The repository's scan history is displayed along with the scan start time, scan status, type of scan (scheduled or manual), link to the pull request if it was created, and the log message if the scan failed. 
+
+. To view your repository on GitHub, click the lick the *Ellipsis* icon (image:ansible-code-bot-dashboard-kebab-icon.png[Ellipsis icon]) beside the repository and select *View repository*.
```
[the remark this answers is not in the document]
Comment (sayjadha) [reply lightspeed/modules/proc_manually-scan-repo.adoc:26]: Typo, good catch! I've made the edit.

**Pull request merged**